### ISSUE TITLE
Main Website; WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by A11Ybot-OP** [istio/istio.io#12022]
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.panel-btn.btn)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Istio - https://istio.io/

Element path: .panel--dark.panel[href$="service-mesh"] > .panel-body > .panel-btn.btn

Snippet: `<span class="btn panel-btn">Learn more</span>`

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.19 (foreground color: #7dc5f2, background color: #3d5080, font size: 13.5pt (18px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 104.0.0.0

**Comment by craigbox** [comment]
Thanks, Mx. Bot!  We'll put this on the backlog, given that (a) web is currently unstaffed, (b) the ratio is close-but-no-cigar (c) the button changes when mouse-over'd.

**Comment by craigbox** [comment]
putting on the backlog for a proposed Q1 2024 site project.

**Comment by craigbox** [comment]
Our site accent color is #7dc6f2 but for some reason this panel is _slightly_ different, with #7dc5f2.  We don't use that color anywhere else.

I would be happy to have let this one slide because (a) the ratio is 4.19 and the target is 4.50 (b) the button changes colour on hover.

If we did want to change it, [this site](https://app.contrast-finder.org/result.html?foreground=%237DC6F2&background=%233D5080&ratio=4.5&isBackgroundTested=false&algo=Rgb&lang=en) suggests we could go with #7dc6f8.

Change is in `_panel.scss`.

**Comment by kfaseela** [comment]
@adilmp1 probably a good issue for you to get started on

**Comment by kfaseela** [comment]
Fixed by @adilmp1
